Toolset/palettes/dictionary/behaviors/revdictionarybehavior.livecodescript

```diff
@@ -93,7 +93,7 @@ end showUpgradeOptions
 
 # bug 17819 enable cmd+c in dictionary
 on commandKeyDown pWhich
-   if pWhich is not "C" then 
+   if pWhich is not "C" and pWhich is not "A" then 
```

Comment:
Could we make this a switch as the logic is quite confusing, and comment the code, something like

switch pWhich 
case "A"
-- trap Cmd + A for select all
case "C"
-- trap Cmd + C for copy
break
case "W"
close me
break
default
pass commandKeyDown
end switch
```

Reply:
Good idea :)